sqlglot/tokens.py

```diff
@@ -930,6 +936,11 @@ class Tokenizer(metaclass=_Tokenizer):
         "USER-DEFINED": TokenType.USERDEFINED,
         "FOR VERSION": TokenType.VERSION_SNAPSHOT,
         "FOR TIMESTAMP": TokenType.TIMESTAMP_SNAPSHOT,
+        # Newly added for Risingwave
+        "SINK": TokenType.SINK,
+        "SOURCE": TokenType.SOURCE,
+        "ENCODE": TokenType.ENCODE,
+        "WATERMARK": TokenType.WATERMARK,
```

Comment:
The encode token is unused, the watermark token is unnecessary (you can do `match_text_seq("WATERMARK")`). You only need `SINK` and `SOURCE` to add them in `CREATABLES` as far as I understand, and you shouldn't include them here -- instead, move them to Risingwave's `Tokenizer.KEYWORDS` mapping.